napari/layers/utils/layer_utils.py

```diff
@@ -233,3 +234,101 @@ def compute_multiscale_level(
     else:
         level = 0
     return level
+
+
+def stack_to_images(stack, axis):
+    """Function to split the active layer into separate layers along an axis
+
+    Parameters
+    ----------
+    stack : napari.layers.Image
+        The image stack to be split into a list of image layers
+    axis : int
+        The axis to split along.
+
+    Returns
+    -------
+    list
+        List of images
+    """
+
+    # if not isinstance(stack, Image):
+    #     print("Active layer is not an image")
+    #     return None
+
+    data = stack.data
+    name = stack.name
+
+    cmaps = list(simple_colormaps.keys())
+    num_dim = len(data.shape)
+
+    if num_dim < 3:
+        print("not enough dimensions")
```

Comment:
A python warning might be more appropriate than a print statement for this sort of thing

https://docs.python.org/3/library/warnings.html